PlayCover/Utils/RestoreGenshinUserData.swift

```diff
@@ -7,12 +7,15 @@
 
 import Foundation
 
-func restoreUserData(folderName: String) {
+func restoreUserData(folderName: String, app: PlayApp) {
+    let BUNDLE_ID = app.info.bundleIdentifier == "com.miHoYo.GenshinImpact"
+                    ? "com.miHoYo.GenshinImpact"
+                    : "com.miHoYo.Yuanshen"
     let accountInfoPlistEncrypt = "MIHOYO_ACCOUNT_INFO_PLIST_2_Encryption"
     let kibanaReportArrayKeyEncrypt = "MIHOYO_KIBANA_REPORT_ARRAY_KEY_Encryption"
     let lastAccountModelEncrypt = "MIHOYO_LAST_ACCOUNT_MODEL_Encryption"
 
-    let gameDataPath = NSHomeDirectory() + "/Library/Containers/com.miHoYo.GenshinImpact/Data/Documents/"
+    let gameDataPath = NSHomeDirectory() + "/Library/Containers/\(BUNDLE_ID)/Data/Documents/"
     let storePath = NSHomeDirectory() + "/Library/Containers/io.playcover.PlayCover/storage/" + folderName + "/"
```

Comment:
Can you update the folder name from `storage` to `Storage` to match the other folders in the Containers folder